sdk/storage/azure-storage-common/src/test/java/com/azure/storage/common/implementation/FluxInputStreamTest.groovy

```diff
@@ -13,6 +13,13 @@ import java.nio.ByteBuffer
 
 class FluxInputStreamTest extends Specification {
 
+    def setup() {
+        String fullTestName = specificationContext.getCurrentIteration().getName().replace(' ', '').toLowerCase()
```

Comment:
Do we still want this?

Reply:
Yeah this is to add the breadcrumbs in live tests so we see them running